main-script.cpp

```diff
@@ -0,0 +1,79 @@
+#include <iostream>
+#include "matrix-matrix-product.hpp"
+#include "matrix-vector-product.hpp"
+#include "read-matrix-market.hpp"
+#include "write-matrix-market.hpp"
+
+using namespace std;
+typedef vector<vector<double>> Matrix;
+typedef vector<double> Vector;
+
+int main(int argc, char* argv[]) {
+    const char* file1 = argv[1];
+    const char* file2 = argv[2];
+    int operation_type = atoi(argv[3]);
```

Comment:
What happens if the user doesn't pass in 3 arguments? Typically we would check if the user passed in enough arguments and print a help message then abort.